build.cake

```diff
@@ -159,8 +160,13 @@ Task("PackBinaries")
 {
     var actions = new List<Action>();
 
-    actions.Add(() => DoPackage("Calamari", "net40", nugetVersion));
+    actions.Add(() => DoPackage("Calamari", "net40", nugetVersion));    
     actions.Add(() => DoPackage("Calamari", "net452", nugetVersion, "Cloud"));
+    
+    // make nuget packages out of Common and CloudAccounts so that they can be used by Sashimi packages
+    actions.Add(() => DoPackage("Calamari.Common", "net40", nugetVersion));
```

Comment:
Do we need to do the same for `netcoreapp3.1`?

Reply:
That's a good question. I haven't checked every Sashimi yet, only Terraform. I'll confirm, and update to include `netcoreapp3.1` if necessary.

Reply:
no need, i know we need it 😉

Reply:
Righto! I'll add that in.

Reply:
It's the netstandard2.1 build target that I've included in this build. (This is what the project was already targeting). 
.NET Core 3.1 supports .NET Standard 2.1 so this should be what you're after. Let me know if I've misunderstood!